src/main/java/in/projecteka/consentmanager/consent/CMProperties.java

```diff
@@ -0,0 +1,19 @@
+package in.projecteka.consentmanager.consent;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConfigurationProperties(prefix = "consentmanager.identifier")
+@Getter
+@Setter
+@NoArgsConstructor(access = AccessLevel.PACKAGE)
+@AllArgsConstructor
+public class CMProperties {
+    private String id;
```

Comment:
Should this ID be same as client registry clientId? Wouldn't it make sense use the same to keep it consistent?

Reply:
Keycloak ClientID?